### ISSUE TITLE
PrintScreen tool when the browser is zoomed (ctrl+)

### ISSUE THREAD

**Issue by alexleduc** [shftcl/www-issues#18]
The extracted image is different of what i saw in the green square

**Comment by nitriques** [comment]
Yeah I do not know how to solve this.. looks crazy http://stackoverflow.com/questions/1713771/how-to-detect-page-zoom-level-in-all-modern-browsers
